operations/src/main/java/com/nextdoor/bender/operation/delete/DeleteFieldOperation.java

```diff
@@ -0,0 +1,32 @@
+package com.nextdoor.bender.operation.delete;
+
+import com.nextdoor.bender.InternalEvent;
+import com.nextdoor.bender.deserializer.DeserializedEvent;
+import com.nextdoor.bender.operation.EventOperation;
+import org.apache.commons.lang3.Validate;
+
+public class DeleteFieldOperation implements EventOperation {
+    private final String key;
+
+    public DeleteFieldOperation(String key) {
+        Validate.notNull(key);
+        this.key = key;
+    }
+
+    @Override
+    public InternalEvent perform(InternalEvent internalEvent) {
+        try {
+            deleteKeyFromEvent(internalEvent);
+        } catch (Exception e) {
+            throw new RuntimeException("JSON field deletion failed.", e);
```

Comment:
Use `OperationException` instead of `RuntimeException`

Reply:
Changed.